src/commands/fun/diceroll.ts

```diff
@@ -0,0 +1,28 @@
+import { Message } from 'discord.js';
+import { Command } from '@lib/types/Command';
+
+export default class extends Command {
+
+	description = 'Get a random number within a user-specified range (min and max inclusive). If no range is specified by the user, defaults to a range from 1 to 6. Also works for negative numbers!';
+	usage = '[min #] | [max #]';
+	aliases = ['random', 'rand'];
+
+	run(msg: Message, [minimum, maximum]: Array<number>): Promise<Message> {
+		return msg.channel.send(`Your random number is ${Math.floor((Math.random() * (maximum - minimum + 1)) + minimum)}`);
+	}
+
+	argParser(_msg: Message, input: string): Array<number> {
```

Comment:
The arg parser should also check that what the user entered is indeed two numbers

Reply:
This should be fixed via the isNaN function and a check for further args using `,...rest`